docs-src/content/using-diatheke-sdk/streaming-asr.md

```diff
@@ -219,4 +248,11 @@ for result in stream.result_stream:
 ```
 {{% /tab %}}
 
+{{% tab "Swift/iOS" %}}
+``` swift
+func handleTTSResponse(_ response: Cobaltspeech_Diatheke_TTSResponse) {
```

Comment:
This function name references TTS, but it is in the docs for ASR. Is this a correct or is it a copy-past error?

Reply:
Yes, that was a copy-past error. Fixed.